README.md

```diff
@@ -12,7 +12,10 @@ Precompiled binaries can be downloaded from [releases](https://github.com/envato
 ### Go
 
 ```
-go get -u github.com/envato/ejsonkms
+go install github.com/envato/ejsonkms@latest
+sudo  ~/go/bin/ejsonkms /usr/local/bin/
```

Comment:
```suggestion
sudo cp ~/go/bin/ejsonkms /usr/local/bin/
```

Reply:
Maybe `${GOBIN:-$HOME/go/bin}` since you don't have to have `GOBIN` set at all.

Reply:
Thanks! I've included both changes